addons/mail/static/src/components/thread_viewer/thread_viewer.scss

```diff
@@ -22,6 +22,7 @@
 
 .o_ThreadViewer_messageList {
     flex: 1 1 auto;
+    min-height: 200px;
```

Comment:
What's the motivation for this? no flicker for composer position being on top of chat window while thread is loading?

Reply:
I think the approach is not to make `message list height at least ...` but instead the attachment list and/or composer height should be controlled.

I pointed out above I had 3 scrollbars and it's confusing, and said I expect 2 instead. But maybe it's okay to have 3 scrollbars but they should be more specific, like scrollbar should not be on composer itself but the attachment list.